app/controllers/project/Projects.scala

```diff
@@ -547,7 +548,8 @@ class Projects @Inject()(stats: StatTracker,
         val data = request.data
         val oldName = data.project.name
         this.projects.rename(data.project, newName).map { _ =>
-          UserActionLogger.log(request.request, UserActionContexts.PROJECT, data.project.id.getOrElse(-1), UserActions.PROJECT_RENAME, s"$author/$newName", s"$author/$oldName")
+          service.insert(UserActionLogEntry(None, None, 1, InetString.apply("::1"), GlobalActionContext.Project, 1, UserActions.VersionDeleted, 1, "1", "2"))
```

Comment:
Any reason for apply call here?

Reply:
debug statement removed in local changes.